frontend/src/constants/goerli-constants.ts

```diff
@@ -0,0 +1,164 @@
+import { ElementAddresses } from "../types/manual/types";
+
+export interface Tranche {
+    expiration: number,
+    address: string,
+    ptPool: {
+        address: string,
+        poolId: string,
+        fee: string,
+        timeStretch: number
+    },
+    ytPool: {
+        address: string,
+        poolId: string,
+        fee: string,
+    },
+    trancheFactory: string,
+    weightedPoolFactory: string,
+    convergentCurvePoolFactory: string,
+}
+
+export const constants: ElementAddresses = {
```

Comment:
wait don't we already have this?
Specifically in ~/constants/goerli-constants.ts?
Curious to know about the replication

Same for other such files.